### DISCUSSION TITLE
Regarding Vulnerability Scan

### DISCUSSION THREAD

**Discussion by rs042009** [apache/airflow#34661] [General]
Hi Team,

What is the tool used to scan the vulnerability for Apache Airflow?  Need inputs on the same.

Thanks,
Rajesh K.

**Comment by hussein-awala** [comment]
In the CI we have a [CodeQL](https://github.com/apache/airflow/blob/main/.github/workflows/codeql-analysis.yml) workflow, triggered after each new commit to main branch.

**Reply by rs042009** [reply]
Thanks
